modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -557,7 +557,9 @@ export default {
     },
     async onSave() {
       if (this.restoreOnly) {
-        await this.restore(this.original);
+        const doc = await this.restore(this.original);
+        this.$emit('modal-result', doc);
+        this.modal.showModal = false;
```

Comment:
restoring content use to unlock the read-only'ness of the doc editor but now your closing this window, this feels like a regression of the experience